userspace/sysdig/plugin_utils.h

```diff
@@ -38,14 +40,70 @@ typedef struct plugin_dir_info
 
 typedef struct
 {
-	string path;
-	string init_config;
-} plugin_selected_info;
+	std::string path;
+	std::string init_config;
+} plugin_selected_init;
 
 void add_plugin_dir(string dirname, bool front_add);
 void add_plugin_dirs(string sysdig_installation_dir);
-void register_plugins(sinsp *inspector);
 std::vector<plugin_dir_info> get_plugin_dirs();
-map<std::string, plugin_selected_info> get_selected_plugins();
-void select_plugin(string& name, const string& init_config = nullptr);
-shared_ptr<sinsp_plugin> enable_plugin(sinsp *inspector, string& name);
\ No newline at end of file
+// Select a plugin for initialization. Name can be either a name or a path.
+void select_plugin_init(string& name, const string& init_config);
+// Select a plugin for use. In case of a source plugin, that the opened one.
```

Comment:
Typo on this comment?

Reply:
No, it's an example of modern art poetry. 😬 The artist wanted to convey the deep meaning of openness.